plugins/c8y_log_plugin/Cargo.toml

```diff
@@ -23,6 +23,7 @@ clap = { version = "3.0", features = ["cargo", "derive"] }
 csv = "1.1"
 easy_reader = "0.5"
 glob = "0.3"
+indoc = "1.0"
```

Comment:
I disagree with the idea to add yet another dependency just to have nice indentation in Rust and the emitted output. 

* The more dependencies, the more issues with outdated ones.
* Having a low footprint is one of our goal.
* Adding a dependency should be done only if there are more benefits than drawbacks.